seqr/views/apis/report_api.py

```diff
@@ -183,8 +183,9 @@ def anvil_export(request, project_guid):
 
 @analyst_required
 def sample_metadata_export(request, project_guid):
-    omit_airtable = project_guid == 'all'
-    if omit_airtable:
+    is_all_projects = project_guid == 'all'
+    omit_airtable = is_all_projects or 'true' in request.GET.get('omitAirtable', '')
```

Comment:
Is it neater to change the condition after the `or` to `request.GET.get('omitAirtable', false)`?

Reply:
not really, it has the same effect and I think differentiating between missing and false is cleaner

Reply:
Or, how about changing it to `'true' === request.GET.get('omitAirtable', '')`. The `in` operation seems to need more computation.

Reply:
No because the param can be an array with a single string